src/scripting/lua_map.cc

```diff
@@ -4739,6 +4807,17 @@ int LuaBuilding::get_flag(lua_State* L) {
  ==========================================================
  */
 
+/* RST
+   .. method:: dismantle()
+
+      Instantly turn this building into a dismantlesite.
+*/
+int LuaBuilding::dismantle(lua_State* L) {
+	Widelands::Building* bld = get(L, get_egbase(L));
+	bld->get_owner()->dismantle_building(bld);
```

Comment:
Use `Game::send_player_dismantle(PlayerImmovable& pi)`?

Reply:
Lua scripts are executed via playercommands, so they can run code directly rather than having to use playercommands. In MP scenario sending a pc from the script would result in the command being sent from *all* players' machines – ouch.

Reply:
OK, we're safe then the way it is :)